skopt/utils.py

```diff
@@ -200,3 +210,62 @@ def func(x):
             best_fun = r.fun
 
     return [v for v in best_x], best_fun
+
+
+def cook_estimator(base_estimator, space=None, **kwargs):
```

Comment:
I dont mind `cook_estimator` :-) 
(This name also kinda imply that we are cooking some black magic inside to come up with a good default, which is nice.)